README.md

```diff
@@ -55,22 +55,26 @@ nextflow run main.nf -entry MODEL_TO_DATA_CHALLENGE -profile local
 
 ```mermaid
 flowchart LR;
-    A[SYNAPSE STAGE]-->D[UPDATE STATUS];
+    A[SYNAPSE STAGE]-->F;
```

Comment:
I updated the workflow so that SYNAPSE STAGE directly links to RUN DOCKER, for 2 reason:

1. RUN DOCKER is the first process that depends on the output of SYNAPSE STAGE (UPDATE STATUS does not, which is what the flowchart previously implied)
2. The flowchart was looking a little weird on the left-hand-side because of the new schematic (CREATE FOLDERS and UPDATE STATUS run concurrently), so this cleaned things up a bit visually.

Let me know if you think this needs more restructuring